### ISSUE TITLE
Lumen 5.4, Call to undefined function OwenIt\Auditing\config_path()

### ISSUE THREAD

**Issue by jcloutz** [owen-it/laravel-auditing#194]
I am receiving the following stack trace in Lumen 5.4 when I try to use artisan for anything.

Have you seen this before?
```
[2017-02-08 18:55:17] lumen.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to undefined function OwenIt\Auditing\config_path() in /Users/jeremycloutier/laravel/lumen-app/vendor/owen-it/laravel-auditing/src/AuditingServiceProvider.php:39
Stack trace:
#0 /Users/jeremycloutier/laravel/lumen-app/vendor/owen-it/laravel-auditing/src/AuditingServiceProvider.php(24): OwenIt\Auditing\AuditingServiceProvider->setupConfig(Object(Laravel\Lumen\Application))
#1 [internal function]: OwenIt\Auditing\AuditingServiceProvider->boot()
#2 /Users/jeremycloutier/laravel/lumen-app/vendor/illuminate/container/BoundMethod.php(28): call_user_func_array(Array, Array)
#3 /Users/jeremycloutier/laravel/lumen-app/vendor/illuminate/support/helpers.php(912): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#4 /Users/jeremycloutier/laravel/lumen-app/vendor/illuminate/container/BoundMethod.php(86): value(Object(Closure))
#5 /Users/jeremycloutier/laravel/lumen-app/vendor/illuminate/container/BoundMethod.php(30): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#6 /Users/jeremycloutier/laravel/lumen-app/vendor/illuminate/container/Container.php(524): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#7 /Users/jeremycloutier/laravel/lumen-app/vendor/laravel/lumen-framework/src/Application.php(176): Illuminate\Container\Container->call(Array)
#8 /Users/jeremycloutier/laravel/lumen-app/bootstrap/app.php(85): Laravel\Lumen\Application->register(Object(OwenIt\Auditing\AuditingServiceProvider))
#9 /Users/jeremycloutier/laravel/lumen-app/artisan(18): require('/Users/jeremycl...')
#10 {main}  
```

**Comment by quetzyg** [comment]
It seems Lumen doesn't have a `config_path()` helper. I'll look into this.

**Comment by quetzyg** [comment]
This has been fixed in PR #197 (version 4.x).
